src/fontStyle.ts

```diff
@@ -0,0 +1,28 @@
+import { createGlobalStyle } from 'styled-components'
+import { theme } from './theme'
+
+export const FontStyle = createGlobalStyle`
+  @font-face {
+    font-family: 'Circular';
+    src: url('https://uploads-ssl.webflow.com/5baa461315ee32413d16236d/5f7f0665430aaedf8cb0bbf3_CircularXXSub-RegularSubset.woff2') format('woff2');
```

Comment:
Are we concerned that this repo is publically accessible and we've put a direct link to download a font in it?